.gitignore

```diff
@@ -4,3 +4,5 @@
 .idea
 /build
 /classes
+out/
```

Comment:
I'm not sure but shouldn't it be "/out"?

Reply:
I picked this one from a template and never had troubles with it before. As for this project, I think both choices are fine.

Reply:
@arkivanov is right. Tried `out/` instead of `/out' and `com.badoo.reaktive.out.Test` is ignored.